tests/legacy/system/test_edb_configuration_2p0.py

```diff
@@ -839,14 +837,14 @@ def test_14_setup_siwave_syz(self, edb_examples):
         }
         edbapp = edb_examples.get_si_verse()
         assert edbapp.configuration.load(data, apply_file=True)
-        data_from_db = edbapp.configuration.get_data_from_db(setups=True)
+        """data_from_db = edbapp.configuration.get_data_from_db(setups=True)
```

Comment:
Same remark as previous comment.

Reply:
This is marked as resolved but the code is still here. Should it be removed ?

Reply:
I thought I removed them. Now they are removed.